src/scripts/check-for-upgrades.ts

```diff
@@ -22,10 +22,15 @@ export class CheckForUpgradesScriptFile extends FileBase {
   }
 
   protected synthesizeContent(resolver: IResolver): string | undefined {
-    const { providerVersion, fqproviderName } = resolver.resolve(
+    let { providerVersion, fqproviderName } = resolver.resolve(
       this.options
     ) as CheckForUpgradesScriptFileOptions;
 
+    // aws -> hashicorp/aws (else the registry API call fails)
+    if (fqproviderName.indexOf("/")) {
```

Comment:
How does this work? From my understanding, the result of indexOf would either be > 0 or < 0, and both would resolve to truthy. Did you mean?

```suggestion
    if (fqproviderName.indexOf("/") < 0) {
```

Reply:
Good catch 😅

Reply:
I think I was intending to use this:
```suggestion
    if (fqproviderName.includes("/")) {
```